ra-tls/libenclave-tls/enclave-tls.h

```diff
@@ -0,0 +1,27 @@
+/* *INDENT-OFF* */
+#ifndef ENCLAVE_TLS_H
+#define ENCLAVE_TLS_H
+/* *INDENT-ON* */
+
+#include <stdbool.h>
+
+// enclave_tls_ctx contains runtime context
+typedef struct enclave_tls_ctx {
+	int sockfd;
+	char *quote_type;	// attestation type
+	bool mutual;		// mutual attestation
+	bool is_server;
+	void* ssl;
```

Comment:
enclave tls should use its own code style rather than wolfssl, so 'type *' instead of 'type* ' is preferred.